File: src/main/java/fr/gouv/stopc/submission/code/server/SubmissionCodeServerApplication.java
package fr.gouv.stopc.submission.code.server;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.context.annotation.ComponentScan;
import org.springframework.scheduling.annotation.EnableAsync;
import org.springframework.scheduling.annotation.EnableScheduling;

import javax.annotation.PostConstruct;

import java.util.TimeZone;

@ComponentScan(basePackages = "fr.gouv.stopc")
@EnableAsync
@EnableScheduling
@SpringBootApplication
public class SubmissionCodeServerApplication {

    public static void main(String[] args) {
        SpringApplication.run(SubmissionCodeServerApplication.class, args);
    }

    @PostConstruct
    void started() {
        // set JVM timezone as UTC
        TimeZone.setDefault(TimeZone.getTimeZone("UTC"));
    }

}

File: submission-code-server-ws-rest-v2/src/main/kotlin/fr/gouv/stopc/submissioncode/SubmissionCodeServerApplication.kt
package fr.gouv.stopc.submissioncode

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.context.properties.ConfigurationPropertiesScan
import org.springframework.boot.runApplication

@SpringBootApplication
@ConfigurationPropertiesScan
class SubmissionCodeServerApplication

fun main(args: Array<String>) {
    runApplication<SubmissionCodeServerApplication>(*args)
}
